packages/module/patternfly-docs/content/extensions/chatbot/examples/Messages/UserMessageWithExtraContent.tsx

```diff
@@ -36,6 +63,304 @@ const BeforeMainContent = () => (
+                OpenShift cluster installation
+              </DescriptionListTerm>
+              <DescriptionListDescription>
+                <Progress
+                  aria-labelledby="title-outside-progress-example-label"
+                  value={progress}
+                  measureLocation={ProgressMeasureLocation.outside}
+                  style={{ '--pf-v6-c-progress--GridGap': 'var(--pf-t--global--spacer--sm' } as React.CSSProperties}
+                  helperText={
+                    <Flex justifyContent={{ default: 'justifyContentSpaceBetween' }}>
+                      <FlexItem>
+                        <HelperText>
+                          <HelperTextItem>
+                            {progress >= 100 ? 'Installation complete!' : getCurrentStageName()}
+                          </HelperTextItem>
+                        </HelperText>
+                      </FlexItem>
+                      <FlexItem>
+                        <HelperText>
+                          <HelperTextItem>{progress >= 100 ? 'Completed' : getTimeRemaining()}</HelperTextItem>
+                        </HelperText>
+                      </FlexItem>
+                    </Flex>
+                  }
+                />
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+          </DescriptionList>
+        </CardHeader>
+        <CardExpandableContent>
+          <CardBody>
+            <hr style={{ marginBlockEnd: 'var(--pf-t--global--spacer--md)' }} />
+            <Accordion>
+              {stages.map((stage) => (
+                <AccordionItem key={stage.id} isExpanded={isAccordionExpanded === stage.id}>
+                  <AccordionToggle
+                    onClick={() => {
+                      onExpandAccordion(stage.id);
+                    }}
+                    id={stage.id}
+                  >
+                    <Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }}>
+                      <FlexItem>{renderStageIcon(stage)}</FlexItem>
+                      <FlexItem>{stage.name}</FlexItem>
+                    </Flex>
+                  </AccordionToggle>
+                  <AccordionContent
+                    id={stage.id.replace('-toggle', '')}
+                    style={
+                      {
+                        '--pf-v6-c-accordion__expandable-content-body--PaddingBlockStart': '0',
+                        '--pf-v6-c-accordion__expandable-content-body--PaddingBlockEnd': '0',
+                        '--pf-v6-c-accordion__expandable-content-body--PaddingInlineStart': '0',
+                        '--pf-v6-c-accordion__expandable-content-body--PaddingInlineEnd': '0',
+                        '--pf-v6-c-accordion__expandable-content--BackgroundColor': 'initial',
+                        '--pf-v6-c-accordion__expandable-content--Color': '#fff'
+                      } as React.CSSProperties
+                    }
```

Comment:
I _think_ you could put these once on this Accordion rather than each time on the content.

Reply:
Seems to work!